### PR TITLE
0.8 port

### PR DESCRIPTION
**Pull request by atotic**

### CODE REVIEW

**Comment by addyosmani** [comment]
Just adding in some of your notes from the other thread: `<google-signin>` is now operational. Got two minor problems:

- no icons display
- no text displays

But, you get rectangles that are functional, and correct color!

The problem is that x-if is not yet evaluating bindings. One more thing, it'll crash for you unless you get my patch for
core-icon. You can work around it in your tree, just a simple check
for null inside `updateIcon()`.

**Comment by addyosmani** [comment]
cc @Zoramite. Looks like we'll be waiting to see what the current status is on `x-if` before this one can proceed further.

**Comment by addyosmani** [comment]
@Zoramite pending further review, we've got a few short term options here. We can either let this work sit in PR until we're happy to switch to 0.8 fully or continue to evolve it via the 0.8-preview branch and land work incrementally. Any thoughts?

**Comment by Zoramite** [comment]
I think I would prefer having it developed out fully in a branch until the entire upstream is ready and we have tested it more.

**Comment by addyosmani** [comment]
SGTM

**Comment by Zoramite** [comment]
I may have misread your earlier comment. I am ok with this being merged into a 0.8-preview branch that we can iterate on. Are there any other changes that need to be made before we merge this into the branch?

File: bower.json [hunk 1]
```diff
@@ -26,14 +26,14 @@
...
-    "google-apis": "GoogleWebComponents/google-apis#~0.4.3",
-    "paper-button": "Polymer/paper-button#~0.5.4",
-    "paper-ripple": "Polymer/paper-ripple#~0.5.4",
-    "paper-shadow": "Polymer/paper-shadow#~0.5.4"
+    "polymer": "Polymer/polymer#0.8-preview",
+    "core-icon": "Polymer/core-icon#0.8-preview",
+    "core-icons": "Polymer/core-icons#0.8-preview",
+    "core-signals": "Polymer/core-signals#0.8-preview",
+    "font-roboto": "Polymer/font-roboto#0.8-preview",
+    "google-apis": "GoogleWebComponents/google-apis0.8-preview",
```
Comment (Zoramite) [inline bower.json]: Is this supposed to have a `#`?

File: demo.html [hunk 2]
```diff
@@ -2,79 +2,79 @@
...
+
+  <title>google-signin Demo</title>
+
+  <script src="../webcomponentsjs/webcomponents-lite.js"></script>
+  <link rel="import" href="google-signin.html">
+  <link rel="import" href="google-signin-aware.html">
+
+  <!-- Demo only styles -->
+  <style>
+    body { font-family: 'RobotoDraft', 'Roboto', sans-serif; line-height:1.2; vertical-align:middle; }
```
Comment (Zoramite) [inline demo.html]: Can we merge these two `body` definitions?

**Comment by addyosmani** [comment]
Let's wait until @atotic has had a chance to review your latest feedback and we can then merge this into a preview branch and iterate from there.

File: bower.json (hunk 1 above)
Comment (atotic) [inline bower.json]: fixed

File: demo.html (hunk 2 above)
Comment (atotic) [inline demo.html]: yes, we can

**Comment by atotic** [comment]
@addyosmani just checked in the fixes. Ready for merge

**Comment by addyosmani** [comment]
:+1: Thanks @atotic!

**Comment by atotic** [comment]
What repository should I check in further fixes? Need to migrate to iron elements

**Comment by addyosmani** [comment]
Same repo, but 0.8-preview branch please.

**Comment by atotic** [comment]
I expected to create branches inside this repository, then generate pull requests for code review. Should I commit to 0.8-preview branch directly?

**Comment by addyosmani** [comment]
Feel free to create branches in this repo that PR back to the 0.8-preview branch for reviews, otherwise I'm fine with directly committing to the 0.8-preview branch unless it's a large set of changes. Interested in @Zoramite's take here.

**Comment by Zoramite** [comment]
I would prefer having pull requests that just target the preview branch. Then we can review all of the changes as they go in instead of trying to review the commit history.

**Pull request merged**